src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -1,9 +1,12 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
- GetContainingTypeOrThis should return null (would be flagged if the compiler was annotated)
- ExplicitInterfaceImplementations would take a null based on the implementation, but not sure if anybody is doing that...

Reply:
> ExplicitInterfaceImplementations would take a null based on the implementation, but not sure if anybody is doing that...

There are some cases where an API would deal with a null input "sanely" but didn't see a need to make it part of the contract, so I omitted it.

Reply:
> GetContainingTypeOrThis should return null (would be flagged if the compiler was annotated)

Fixed. 

Separately, I have a local annotated copy of `ISymbol` and can say we're gonna have a fun time propagating that change. 😄